### PR TITLE
Updated GraphQL Go tutorial

### PR DESCRIPTION
**Pull request by ashwamegh**
Updates:
- Example code has been fixed to be executed correctly to avoid errors on the user's side
- Fixes typo errors in the tutorial
- Adds the missing step in database tutorial (to create database before migration)
- Changed `golang-migrate` command to use the working release (latest release is causing errors while running server)
- Updated the example code to same variables everywhere to maintain consistency

Addresses #1094

### CODE REVIEW

**Comment by Glyphack** [comment]
Hi @ashwamegh Thanks for your contribution. I will review it within two days and update the tutorial code repo.

**Comment by ashwamegh** [comment]
@Glyphack Thanks 👍👍

**Review by Glyphack — APPROVED** [review]
LGTM

File: content/backend/graphql-go/1-getting-started.md [hunk 1]
```diff
@@ -33,6 +34,9 @@ Here is a description from gqlgen about the generated files:
 Now let's start with defining schema we need for our API.
 We have two types Link and User each of them for representing Link and User to client, a `links` Query to return list of Links. an input for creating new links and mutation for creating link. we also need mutations to for auth system which includes Login, createUser, refreshToken(I'll explain them later) then run the command below to regenerate graphql models.
 ```js
+// graph/schema.graphqls
```
[the remark this answers is not in the document]
Comment (ashwamegh) [reply content/backend/graphql-go/1-getting-started.md:37]: okay, let me update it

**Review by Glyphack — APPROVED** [review]

**Comment by ashwamegh** [comment]
Can we merge this PR @anku255 @Glyphack @nikolasburk

**Comment by nikolasburk** [comment]
Thanks a lot @ashwamegh 🎉

**Pull request merged**